src/Compilers/CSharp/Portable/Emitter/Model/PEAssemblyBuilder.cs

```diff
@@ -168,23 +168,27 @@ protected override void AddEmbeddedResourcesFromAddedModules(ArrayBuilder<Cci.Ma
 
         internal override SynthesizedAttributeData SynthesizeEmbeddedAttribute()
         {
-            if ((object)_lazyEmbeddedAttribute != null)
-            {
-                return new SynthesizedAttributeData(
-                    _lazyEmbeddedAttribute.Constructor,
-                    ImmutableArray<TypedConstant>.Empty,
-                    ImmutableArray<KeyValuePair<string, TypedConstant>>.Empty);
-            }
+            return new SynthesizedAttributeData(
+                _lazyEmbeddedAttribute.Constructors[0],
```

Comment:
>_lazyEmbeddedAttribute [](start = 16, length = 22)

Is it still possible for `_lazyEmbeddedAttribute` to be `null`?

Reply:
That would be a bug. Previously, we were relying on `base.SynthesizeEmbeddedAttribute()` which threw `ExceptionUtilities.Unreachable`.

Reply:
🎉 
Same may apply to method below (`SynthesizeNullableAttribute`).

Reply:
From discussion with Chuck, we need to decide if `NullableAttribute` should only be embedded, or whether we could add it to frameworks at some point.